sdk/identity/Azure.Identity/tests/SessionRecords/UsernamePasswordCredentialLiveTests/AuthenticateUsernamePasswordLive.json

```diff
@@ -197,18 +199,17 @@
       }
     },
     {
-      "RequestUri": "https://login.microsoftonline.com/common/userrealm/tempuser2@azuresdkplayground.onmicrosoft.com?api-version=1.0",
+      "RequestUri": "https://login.microsoftonline.com/common/userrealm/testuser@azuresdkplayground.onmicrosoft.com?api-version=1.0",
```

Comment:
Despite being a test user, should secrets be sanitized? I have some code I wrote in Search that should work with some adaptation. Doesn't need to deserialize the object - just looks for secret test based on variables, though I'm not sure the latter part works in this case.

Reply:
The secrets are sanitized, I suppose the username would be considered PII, but since it's test account I'm not sure that privacy is a concern.